sdks/java/core/build.gradle

```diff
@@ -105,5 +105,6 @@ dependencies {
   shadowTest library.java.avro_tests
   shadowTest library.java.zstd_jni
   shadowTest library.java.jamm
+  shadowTest library.java.xz_java
```

Comment:
Why do we need to add this?

Reply:
Because starting from Avro 1.9.0 this dependency became `provided` but we need this for tests